src/jupyter_contrib_nbextensions/nbextensions/toc2/toc2.yaml

```diff
@@ -52,4 +52,19 @@ Parameters:
   input_type: color
   description: Color of sections with running cells
   default: "#FF0000"  
-
+- name: toc2.colors.wrapper_background
+  input_type: color
+  description: Color of wrapper window background
+  default: "#FFF"
```

Comment:
I have a feeling that defaults specified as three-character codes don't load well in the configurator - could we use full six-char codes here?